tests/bsim/bluetooth/audio/prj.conf

```diff
@@ -6,6 +6,7 @@ CONFIG_BT=y
 CONFIG_BT_CENTRAL=y
 CONFIG_BT_PERIPHERAL=y
 CONFIG_BT_DEVICE_NAME="bsim_test_audio"
+CONFIG_BT_EATT=y
```

Comment:
You probably also need to add `CONFIG_BT_L2CAP_ECRED=y` for this to take effect, right?